<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>platform</class>
 <widget class="QWidget" name="platform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1276</width>
    <height>724</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1276</width>
    <height>724</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1276</width>
    <height>724</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1276</width>
     <height>724</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文新魏</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{border-image: url(:/Picture/platform.jpg);border-top-left-radius:15px;border-top-right-radius:15px;border-bottom-left-radius:15px;border-bottom-right-radius:15px;}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>620</y>
      <width>151</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文新魏</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255,255,255);border: 3px solid white;border-radius:8px;color:rgb(74,112,133);</string>
    </property>
    <property name="text">
     <string>退出登录</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>190</y>
      <width>741</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="user_name">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文新魏</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="location_edit">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>30</y>
      <width>231</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="room_type">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>120</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文新魏</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>全部</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>单人间</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>标准间</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>豪华间</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>商务间</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="rate">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>110</y>
      <width>241</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文新魏</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>优先高评价</string>
    </property>
   </widget>
   <widget class="QPushButton" name="my_order">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>620</y>
      <width>151</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文新魏</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255,255,255);border: 3px solid white;border-radius:8px;color:rgb(74,112,133);</string>
    </property>
    <property name="text">
     <string>我的订单</string>
    </property>
   </widget>
   <widget class="QPushButton" name="order">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>620</y>
      <width>151</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文新魏</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255,255,255);border: 3px solid white;border-radius:8px;color:rgb(74,112,133);</string>
    </property>
    <property name="text">
     <string>查看房间</string>
    </property>
   </widget>
   <widget class="QPushButton" name="query">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>620</y>
      <width>151</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文新魏</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255,255,255);border: 3px solid white;border-radius:8px;color:rgb(74,112,133);</string>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="city_edit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>231</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="order_type">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>120</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文新魏</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAtCurrent</enum>
    </property>
    <item>
     <property name="text">
      <string>升序</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>降序</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string> City:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string> Price:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>30</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string> Area:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>110</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string> Room:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
